com.system76.CosmicBaseApp.json

```diff
@@ -0,0 +1,63 @@
+{
+    "id": "com.system76.CosmicBaseApp",
+    "branch": "1.0.0-alpha.2",
```

Comment:
Does these components have an API/ABI involved ? Or apps are supposed to always use the latest version of theme from here ?

If so I'd drop the `branch`.

Reply:
There are no ABI/API versioned components here as of now.

Reply:
I have removed `branch`